lib/swaps/Swaps.ts

```diff
@@ -57,7 +57,7 @@ type SwapDeal = {
   makerToTakerRoutes?: lndrpc.Route[];
   createTime: number;
   executeTime?: number;
-  competionTime?: number
+  completionTime?: number
```

Comment:
Can we call this `completeTime`? Just to be consistent with `createTime` and `executeTime` above.